browser/component_updater/brave_component_updater_configurator.cc

```diff
@@ -22,10 +22,12 @@
 #include "components/prefs/pref_registry_simple.h"
```

Comment:
cc @jumde please review this commit: "Change to classes inheriting from update_client::Configurator."